test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -686,6 +686,25 @@ class test_ucp_sockaddr : public ucp_test {
                      ucp_ep_config_lane_is_peer_match(key1, lane1, key2, lane2)));
     }
 
+    bool
+    has_rndv_lanes_use_p2p_not_support_md_invalidate(ucp_ep_h ep)
```

Comment:
1. function name too long
2. can we unify the check with the code in check_rndv_lanes?

Reply:
done